### DISCUSSION TITLE
Can you specify the scope of other Documents, because I don't want to have articles embedded in articles

### DISCUSSION THREAD

**Discussion by blueMountain007** [payloadcms/payload#1281] [Q&A] [answered]
###  Can you specify the scope of other Documents, because I don't want to have articles embedded in articles
<img width="814" alt="image" src="https://user-images.githubusercontent.com/16400882/196903042-040d91bd-8faf-49f2-8122-5ca32642eb93.png">

<img width="840" alt="image" src="https://user-images.githubusercontent.com/16400882/196903096-93ff4c03-5467-4480-a6d8-36d071ac1e4c.png">

[https://payloadcms.com/docs/fields/rich-text#relationship-element](https://payloadcms.com/docs/fields/rich-text#relationship-element)

**Comment by jmikrut** [comment, marked as answer]
Yes, you can use the `admin.enableRichTextRelationship` property. Set it to false for any collection you don't want to be able to link to.  

Right now, rich text uploads offer a bit more granularity here, as you can specifically whitelist collections to be able to select from on a field-by-field basis. But in the future we should be able to add more granularity to do this with rich text relationships too. 👍
